tests/flow/test_function_calls.py

```
@@ -2438,375 +2438,365 @@ def test87_typeof(self):
+
+        # NULL input should return NULL
+        expected_result = [None]
+        query = """WITH NULL as string RETURN string.replaceRegEx("bla", "bla", null)"""
+        actual_result = self.graph.query(query)
+        self.env.assertEquals(actual_result.result_set[0], expected_result)
+
+        # test invalid regex
+        try:
+            query = """RETURN string.replaceRegEx('aa', '?')"""
+            self.graph.query(query)
+        except ResponseError as e:
```

Ruff (0.14.13)

`ResponseError` may be undefined, or defined from star imports

(F405)